options/options.go

```diff
@@ -80,6 +80,9 @@ type TerragruntOptions struct {
 	// exposed here primarily so we can set it to a low value at test time.
 	MaxFoldersToCheck int
 
+	// Unix-style glob of directories to exclude when running *-all commands
+	ExcludeDir string
```

Comment:
Having looked through the code, I think `ExcludeDirs []string` would be a better fit.

```
terragrunt apply-all --terragrunt-exclude-dir foo --terragrunt-exclude-dir bar/*
```

Reply:
Renamed the variable.